utils/sqlite_manager.py

```diff
@@ -1,37 +1,56 @@
 import os
-import streamlit as st
-
+import json
 import pandas as pd
+from enum import Enum
 from sqlalchemy import create_engine, text
 from sqlalchemy.orm import sessionmaker
 
-from utils.data_manipulation import StrategyData
+from utils.data_manipulation import StrategyV1Data
 
 
-class DatabaseManager:
-    def __init__(self, db_name: str, executors_path: str = "data"):
-        self.db_name = db_name
-        # TODO: Create db path for all types of db
-        self.db_path = f'sqlite:///{os.path.join(db_name)}'
-        self.engine = create_engine(self.db_path, connect_args={'check_same_thread': False})
-        self.session_maker = sessionmaker(bind=self.engine)
+class CloseType(Enum):
+    TIME_LIMIT = 1
+    STOP_LOSS = 2
+    TAKE_PROFIT = 3
+    EXPIRED = 4
+    EARLY_STOP = 5
+    TRAILING_STOP = 6
+    INSUFFICIENT_BALANCE = 7
+    FAILED = 8
+    COMPLETED = 9
 
-    def get_strategy_data(self, config_file_path=None, start_date=None, end_date=None):
-        def load_data(table_loader):
-            try:
-                return table_loader()
-            except Exception as e:
-                return None  # Return None to indicate failure
 
-        # Use load_data to load tables
-        orders = load_data(self.get_orders)
-        trade_fills = load_data(self.get_trade_fills)
-        order_status = load_data(self.get_order_status)
-        market_data = load_data(self.get_market_data)
-        position_executor = load_data(self.get_position_executor_data)
+class SQLiteManager:
```

Comment:
Change name of objects that instead of describing the technology used, describes what represents in the system

Reply:
eg. HummingbotDatabase